README.md

```diff
@@ -1289,6 +1289,27 @@ Other Style Guides
 
   - [15.4](#15.4) <a name='15.4'></a> For more information see [Truth Equality and JavaScript](http://javascriptweblog.wordpress.com/2011/02/07/truth-equality-and-javascript/#more-2108) by Angus Croll.
 
+  - [15.5](#15.5) <a name='15.5'></a> Ternaries should not be nested and generally be single line expressions.
```

Comment:
They can't optimize it because `toString` can have side effects - but looking up and stringifying a few properties really isn't that big a deal. If it was, then you'd move those things to a function, and it'd still be a one-line ternary.